apps/services/handlers.py

```diff
@@ -116,15 +176,39 @@ async def cancel_request(university_id: int, user_request_id: int, cancel_reques
             "user_request_id": user_request_id,
             "status_id": cancel_request.status_id
         },
-        "message": f"Canceled request with id {user_request_id}"
+        "message": f"Canceled request with id {user_request_id}",
+        "code": http_status.HTTP_202_ACCEPTED
     }
 
 
 @services_router.post("/{university_id}/user-request-review/{user_request_id}/",
+                      name="create_user_request_review",
                       response_model=JSENDOutSchema[UserRequestReviewOut],
+                      summary="Create user request review",
+                      responses={200: {"description": "Successful create user request review response"}},
                       tags=["Admin dashboard"])
 async def create_user_request_review(university_id: int, user_request_id: int, user_request_review: UserRequestReviewIn,
                                      user=Depends(get_current_user)):
+    """
+    **Create user request review**
+
+    **Path**:
+    - **university_id**: user university id
+    - **user_request_id**: user request id
+
+    **Input**:
+    - **status_id**: user request status id, required
+    - **room_number**: user room number, required
+    - **start_date_accommodation**: starting datetime hostel accommodation, required
+    - **end_date_accommodation**: end datetime hostel accommodation, required
+    - **total_sum**: total sum of hostel accommodation payment, required
+    - **payment_deadline**: deadline datetime for hostel accommodation payment, required
+    - **remark**: additional info for request review, required
+    - **hostel_id**: hostel id in the database, required
+    - **bed_place_id**: hostel bed place id, required
```

Comment:
more of these fields aren't required

Reply:
I've asked about it Konstantin, he said that we can add such info in cases with a great amount different data, that need to be described. Of course I can delete it, but first would be a good idea to see how it looks. Waiting for further info what to do.

Reply:
ok ,ok for now